content/chainguard/chainguard-images/network-requirements.md

```diff
@@ -64,3 +64,23 @@ You will need egress rules that allow new traffic to the hosts listed here. You
 Many of the hosts listed on this page use multiple DNS A records or CNAME aliases. Additionally, many A records have a short time to live of 60 seconds, and the majority are less than an hour (3600s).
 
 If your network filters traffic based on IP addresses, ensure that any firewalls update their rules at an appropriate interval to match the TTL for each DNS record.
+
+## Minimum TLS requirements
+
+For guaranteed connectivity, the following TLS requirements must be at
+minimum supported by clients/servers communicating with Chainguard
```

Comment:
```suggestion
minimum supported by clients and servers communicating with Chainguard
```

Reply:
I know servers are basically clients in this context so you could also leave "servers"  out

Reply:
We have 3 connectivity cases

* customer client connecting to -> [chainguard container as a server]
* chainguard container client connecting to -> [customer hosted server]
* customer clients connecting -> [chainguard host endpoints cgr.dev etc]

In above pairs, both sides are all unique, and these days all languages actually have different ciphersuites they offer and support as client and as server.

For example go supports TLSv1.0 & TLSv1.1 as a client; but not as a server.
Similarly OpenSSL too has configs to expose different ciphers as a client and as a server.

So it is a mess.

Reply:
Yeah .. I am just aware of it a bit more in the Java side.. but in the wording here we can therefore stick to "clients and servers"